LeaderboardBackend/Models/Requests/UserRequests.cs

```diff
@@ -97,14 +90,15 @@ public record RegisterRequest
     ///     </ul>
     /// </summary>
     /// <example>P4ssword</example>
-    [Password]
-    [Required]
-    public string Password { get; set; } = null!;
+    public required string Password { get; set; }
+}
 
-    /// <summary>
-    ///     The password confirmation. This value must match `Password`.
-    /// </summary>
-    [Compare("Password", ErrorMessage = "The password confirmation must match your password.")]
-    [Required]
-    public string PasswordConfirm { get; set; } = null!;
```

Comment:
Why has this been removed?

Reply:
Ensuring that the user correctly typed their password is a client-side concern.